http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -63,14 +66,35 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
         case x: RuntimeConfig.AzureConfig => F.pure(x)
         case x => F.raiseError(new RuntimeException(s"this runtime doesn't have proper azure config ${x}"))
       }
+      leoAuth <- samDAO.getLeoAuthToken
+      workspaceDescOpt <- wsmDao.getWorkspace(msg.workspaceId, leoAuth)
+      workspaceDesc <- F.fromOption(workspaceDescOpt, WorkspaceNotFoundException(msg.workspaceId, ctx.traceId))
+      storageContainerOpt <- wsmDao.getWorkspaceStorageContainer(msg.workspaceId, leoAuth)
```

Comment:
Maybe minor point -- but I think we're already passing the storage container in the `CreateAzureRuntimeMessage`? We could probably just construct the `workspaceStorageContainerUrl` with information we have instead of making 3 new calls to WSM. Something like:

` s"https://${params.landingZoneResources.storageAccountName.value}.blob.core.windows.net/${params.storageContainer.name.vale}"

See also: https://learn.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata#resource-uri-syntax